go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/DataDog/datadog-agent
 
-go 1.17
+go 1.18
```

Comment:
You also need to update the `.go-version` at the root, with `1.18.5`

Reply:
Huh, I think that got reverted somehow.